.pre-commit-config.yaml

```diff
@@ -12,7 +12,6 @@ repos:
     hooks:
       - id: go-fmt
       - id: go-vet
-      - id: go-imports
```

Comment:
go-imports does not fully support go 1.18.
removing for now. Go fmt is good enough.